thriftrw/spec/struct.py

```diff
@@ -459,7 +459,10 @@ def struct_cls(struct_spec, scope):
 
         default = None
         if field.default_value is not None:
-            default = field.default_value
+            if field.ttype_code == TType.BINARY:
+                default = field.default_value.encode('utf-8')
+            else:
+                default = field.default_value
```

Comment:
You're changing this for structs only. Const binary fields on unions and exceptions won't work.